### ISSUE TITLE
[Feature request] Option to disable bright for bold

### ISSUE THREAD

**Issue by xsrvmy** [lxqt/qterminal#883]
TL;DR: Add an option to render SGR escape code 1 using regular instead of bright colors.

It seems that more and more terminal emulators these days are providing an option for not using bright colors for bold (SGR 1). Some terminal emulators (eg. kitty) don't even allow using bright for bold. The basic argument is that SGR 1 making colors brighter was only supposed to be used in cases when the terminal is incapable of displaying bold text (similar to how tty sometimes uses cyan for italic), and not in modern terminals that can use bold. Indeed, some modern TUI programs are using bright colors (8-15) without using the bold color. It would be nice if qterminal offered an option for not switching to bright colors for bold.

Note that there is a toggle right now for using bold for bright colors. This is the opposite of this and is confusingly worded (it probably should use "intense", bright colors could mean escape codes 90-97 which are not bold anyways), and it does the opposite of what is being requested.

Also I am aware that this is probably something inherited from Konsole. However, I am not sure how much qterminal is tied to konsole these days, so that's why I am reporting this issue here. Qterminal seems to be 2x faster after all.

**Comment by kolalok** [comment]
I too would appreciate such an option, it would make the Solarized themes look right.

It seems to be controlled by [this line](https://github.com/lxqt/qtermwidget/blob/674008600c6f22621c6694362809ed019eca05b2/lib/Screen.cpp#L453) in qtermwidget. I recompiled qtermwidget with this block commented out and now the colors are to my liking. I don't however feel up to designing a proper UI control.
